### PR TITLE
Added Rust and Motoko examples to hello

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/motoko_examples/hello/canisters/motoko/main.mo [hunk 1]
```diff
@@ -0,0 +1,7 @@
+actor {
+
+  public func greet(name : Text) : async Text {
```
Comment (lastmjs) [inline examples/motoko_examples/hello/canisters/motoko/main.mo:3]: Should this be a query function?

File: examples/motoko_examples/hello/canisters/rust/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#[ic_cdk_macros::query]
+pub fn greet(name: String) -> String {
+    format!("Hello, {name}!")
```
Comment (lastmjs) [inline examples/motoko_examples/hello/canisters/rust/src/lib.rs:3]: This works? I didn't think format! worked like this

File: examples/motoko_examples/hello/dfx.json [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
             "frontend": {
-                "entrypoint": "src/hello_assets/src/index.html"
+                "entrypoint": "canisters/hello_assets/src/index.html"
             },
-            "source": ["src/hello_assets/assets", "dist/hello_assets/"],
+            "source": ["canisters/hello_assets/assets", "dist/hello_assets/"],
             "type": "assets"
+        },
+        "motoko": {
+            "main": "canisters/motoko/main.mo",
```
Comment (lastmjs) [inline examples/motoko_examples/hello/dfx.json:22]: type: motoko

File: examples/motoko_examples/hello/canisters/motoko/main.mo [hunk 4]
```diff
@@ -0,0 +1,7 @@
+actor {
```
Comment (lastmjs) [inline examples/motoko_examples/hello/canisters/motoko/main.mo:1]: Has this been added to the benchmarks.yml?

File: examples/motoko_examples/hello/canisters/motoko/main.mo [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (lastmjs) [inline examples/motoko_examples/hello/canisters/motoko/main.mo:2]: Does the frontend still work?

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: examples/motoko_examples/hello/canisters/motoko/main.mo (hunk 1 above)
Comment (bdemann) [reply examples/motoko_examples/hello/canisters/motoko/main.mo:3]: That's how it is the motoko examples https://github.com/dfinity/examples/blob/master/motoko/hello/src/hello/main.mo
Should we change all of them to be updates?

File: examples/motoko_examples/hello/canisters/rust/src/lib.rs (hunk 2 above)
Comment (bdemann) [reply examples/motoko_examples/hello/canisters/rust/src/lib.rs:3]: Yes, I think it's how it does it in the default rust canister

File: examples/motoko_examples/hello/canisters/motoko/main.mo (hunk 5 above)
Comment (bdemann) [reply examples/motoko_examples/hello/canisters/motoko/main.mo:2]: yes

**Review by lastmjs — APPROVED** [review]

**Pull request merged**